app/src/main/res/layout/activity_main.xml

```diff
@@ -4,14 +4,10 @@
     android:id="@+id/activity_main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingBottom="@dimen/activity_vertical_margin"
-    android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
-    android:paddingTop="@dimen/activity_vertical_margin"
     tools:context="com.sdsmdg.hareshkh.lectureassignment.MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Good luck with your assignments" />
+    <android.support.v7.widget.RecyclerView
```

Comment:
RecyclerView is an extension of a view group, and can be used as root layout in absence of other views, which is the case here.